xml/System.Net.Security/TlsCipherSuite.xml

```diff
@@ -5019,7 +5026,7 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Represents the TLS_NULL_WITH_NULL_NULL cipher suite; this is the default value.</summary>
```

Comment:
@krwq this might worth reviewing. In this field I am stating `TLS_NULL_WITH_NULL_NULL` is enum's default value; not sure if it should really be included.

Reply:
it is default when you do `default(TlsCipherSuite)` solely because enums always default to 0 which happens to map to this cipher suite. For the AllowedCipherSuites you will get whatever you pass in and there is only `custom` or `OS default` options